config/environments/production.rb

```diff
@@ -1,164 +1,86 @@
-  config.active_support.deprecation = :notify
+  # Disable Rails's static asset server (Apache or nginx will already do this).
+  config.serve_static_assets = true
 
-  # Compress JavaScripts and CSS
+  # Compress JavaScripts and CSS.
   config.assets.js_compressor = :uglifier
+  # config.assets.css_compressor = :sass
 
-  # Fallback to assets pipeline if a precompiled asset is missed:
+  # Do not fallback to assets pipeline if a precompiled asset is missed.
   # that's the case when an engine with it's own assets is added to Foreman later in production.
   config.assets.compile = false
 
-  # Generate digests for assets URLs
+  # Generate digests for assets URLs.
   config.assets.digest = true
 
-  # Defaults to Rails.root.join("public/assets")
-  # config.assets.manifest = YOUR_PATH
+  # `config.assets.precompile` has moved to config/initializers/assets.rb
+
+  # Specifies the header that your server uses for sending files.
+  # config.action_dispatch.x_sendfile_header = "X-Sendfile" # for apache
+  # config.action_dispatch.x_sendfile_header = 'X-Accel-Redirect' # for nginx
 
   # Should ANSI color codes be used when logging information
   config.colorize_logging = Foreman::Logging.config[:colorize]
 
-  # Add the fonts path
-  config.assets.paths << Rails.root.join('vendor', 'assets', 'fonts')
-
-  # Precompile additional assets
-  config.assets.precompile << /\.(?:svg|eot|woff|ttf)$/
-
-  # Precompile additional assets (application.js, application.css, and all non-JS/CSS are already added)
-  #  config.assets.precompile += %w()
-  #
-  javascript = %w(compute_resource
-                  compute_resources/libvirt/nic_info
-                  compute_resources/ovirt/nic_info
-                  compute_resources/vmware/nic_info
-                  lookup_keys
-                  editor
-                  diff
-                  host_edit
-                  host_edit_interfaces
-                  hosts
-                  jquery.cookie
-                  host_checkbox
-                  nfs_visibility
-                  noVNC/base64
-                  noVNC/des
-                  noVNC/display
-                  noVNC/input
-                  noVNC/jsunzip
-                  noVNC/logo
-                  noVNC/playback
-                  noVNC/rfb
-                  noVNC/ui
-                  noVNC/util
-                  noVNC/websock
-                  noVNC/webutil
-                  noVNC
-                  reports
-                  spice
-                  trends
-                  charts
-                  taxonomy_edit
-                  gettext/all
-                  filters
-                  users
-                  class_edit
-                  dashboard
-                  auth_source_ldap
-                  subnets
-                  hidden_values
-                  password_strength)
-
-  stylesheets = %w( )
-
-  config.assets.precompile += javascript.map{|js| js + ".js"} + stylesheets + %w(background-size.htc)
```

Comment:
they are moved to a separate file as rails by default now uses an assets initializer.